### PR TITLE
django-cheatsheet

### PR DESCRIPTION
**Pull request by rsunder10**
https://duck.co/ia/view/django_cheatsheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@rsunder10 Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/django.json [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+			"val": "to display usage information and a list of the commands provided by each application."
+		}, {
+			"key": "django-admin help --commands",
+			"val": "to display a list of all available commands."
+		}, {
+			"key": "django-admin help <command>",
+			"val": "to display  description of command"
+		}, {
+			"key": "django-admin version",
+			"val": "to show version "
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/django.json]: Remove the `to` so it reads `show version` - do this for the others too.

Comment (rsunder10) [inline share/goodie/cheat_sheets/json/django.json]: ok i will do it

File: share/goodie/cheat_sheets/json/django.json [hunk 2]
```diff
@@ -0,0 +1,266 @@
+
+{
+	"id": "django_cheat_sheet",
+	"name": "Django Cheatsheet",
+	"description": "To list the Django Mostly Used Functionality .Which will be beneficial for the Extensive Django Users/Developers ",
+
+	"metadata": {
+        "post":"https://duck.co/ia/view/django_cheatsheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/django.json]: Please remove this, `post` is not a valid property

Comment (rsunder10) [inline share/goodie/cheat_sheets/json/django.json]: removed

File: share/goodie/cheat_sheets/json/django.json [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+			"key": "django-admin help --commands",
+			"val": "display a list of all available commands."
+		}, {
+			"key": "django-admin help <command>",
+			"val": "display  description of command"
+		}, {
+			"key": "django-admin version",
+			"val": "show version "
+		}, {
+			"key": "django-admin check ",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/django.json]: Lose the space at the end of the key - likewise for others.

File: share/goodie/cheat_sheets/json/django.json [hunk 4]
```diff
@@ -0,0 +1,265 @@
+
+{
+	"id": "django_cheat_sheet",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/django.json]: Oh my word, the tabs!

Please get rid of the tabs at the line start.

**Comment by GuiltyDolphin** [comment]
@pjhampton Would you mind reviewing this one? I imagine your knowledge with Django exceeds my own!

I noticed #2779 - anything you want to bring from there?

**Comment by pjhampton** [comment]
@GuiltyDolphin sure.

@rsunder10 consider deleting **QuerySet Commands **because there is no context ~ It doesn't make much of a contribution to a cheat sheet. Check out the one I was developing here for some ideas - feel free to take stuff (#2779). The next time you update your PR it should override mine on the beta server.

**Comment by moollaza** [comment]
@rsunder10 thanks again for creating the Django Cheat Sheet! You’ve been doing a really good job here and we'd like you to help make DuckDuckGo even better for yourself and others interested in Django.

When you're ready to move on, please: 

- Create a new Instant Answer for searches you do related to Django

OR

- Test live Django Instant Answers (or those in development) for bugs and suggest improvements

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Creating-or-Improving-Instant-Answers).

Thanks again!

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=django cheatsheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/django_cheatsheet_mobile.png)

**Comment by moollaza** [comment]
@rsunder10 Thanks! This LGTM :+1:

It looks like @GuiltyDolphin has some feedback about improvements that we'd like to see implemented (correcting tabs)

**Comment by daxtheduck** [comment]
@rsunder10 this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Comment by moollaza** [comment]
@rsunder10 I would also suggest looking at @pjhampton 's PR and using it to improve this one 👍

**Pull request merged**